flexget/plugin.py

```diff
@@ -416,13 +412,19 @@ def _load_plugins_from_dirs(dirs):
                       'point (before, after). Plugin is not working properly.' % (args[0], phase))
 
 
-def load_plugins():
-    """Load plugins from the standard plugin paths."""
+def load_plugins(extra_dirs=[]):
```

Comment:
We can't use a mutable default argument (a list is mutable), as it won't reset if the function is called more than once. Usually, you'd set the default to `None`, then create your list in the body of the function.